website/pages/en/index.js

```diff
@@ -415,7 +415,8 @@ const UsersSection = ({ language }) => {
             </a>
             <div style={{ marginLeft: ".7em", width: "260px" }}>
               <p>
-                More than <strong>3000</strong> tools and integrations on npm
+                More than <strong data-name="dependent-npm">5.21 k</strong>{" "}
```

Comment:
So `data-placeholder` or `data-name`? Also "More than 5000" would look less strange than "More than 5.21 k", IMHO.

Reply:
I use `data-placeholder` first... I don't know why I choose `data-name` when commit. 😄 

`human-format` format it this way

Reply:
5.21 k looks like a file size :)

Reply:
[human-number](https://npm.runkit.com/human-number) `5210` -> `5.21K` `1639226` -> `1.639226M` ...

Reply:
[hrn](https://runkit.com/embed/2lw28ebcp84x)  `["5 K", "1 M"]`